index.js

```diff
@@ -119,7 +120,7 @@ function sendToIngest(event, context, aimsC, collectedBatch, callback) {
         if (compressionErr) {
             return callback(compressionErr);
         } else {
-            var ingest = new m_alServiceC.IngestC(INGEST_ENDPOINT, aimsC);
+            var ingest = new m_alServiceC.IngestC(INGEST_ENDPOINT, aimsC, 'lambda_function');
```

Comment:
Ingestc doesn't care about collector type. I don't this we want to maintain collector type -> function type mapping in Ingestc and update it every time we add we add new collector type.
Azcollect is the only service (most likely) which needs collector type because it is a part of azcollect URL.